x11-misc/arandr/arandr-0.1.10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 python3_7 )
+inherit distutils-r1
+
+DESCRIPTION="Another XRandR GUI"
+HOMEPAGE="https://christian.amsuess.com/tools/arandr/"
+SRC_URI="https://christian.amsuess.com/tools/${PN}/files/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-python/pygobject-3[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	>=dev-python/docutils-0.6[${PYTHON_USEDEP}]
+	x11-apps/xrandr
```

Comment:
Dependency lists should be sorted. This makes it easier to find/see a particular dependency. I just let my editor sort the list (it will put version constraints first, which is what you'll typically find in other packages).